nltk/tokenize/destructive.py

```diff
@@ -113,6 +114,15 @@ class NLTKWordTokenizer(TokenizerI):
     CONTRACTIONS3 = list(map(re.compile, _contractions.CONTRACTIONS3))
 
     def tokenize(self, text, convert_parentheses=False, return_str=False):
+
+        if return_str is not False:
```

Comment:
```suggestion
        if return_str:
```
Well spotted @12mohaned! This is also more in line with the old behaviour of `text if return_str else text.split()`.